proselint/tools.py

```diff
@@ -31,43 +32,36 @@ def line_and_column(text, position):
 
 
 def lint(
-    input_file: Union[str,
-    FileIO],
+    input_file: Union[str, FileIO],
```

Comment:
```suggestion
    input: Union[str, FileIO],
```
Just think this would be neater, `input` isn't guaranteed to be a file and it's more generic
This is assuming there is programmatic capability and it's not just redundant

Reply:
`source` could also work